css/rtk.css

```diff
@@ -0,0 +1,28 @@
+#rikaichan-window .kpr-story b,strong{
+     color:black !important;
+     font-weight: bold !important;      
+}
+
+#rikaichan-window .kpr-story i,em{
```

Comment:
why manually style these tags instead of using default or just using your own classes?